python/py-logcat-color/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
+PortGroup           github 1.0
+
+set _name           logcat-color
+
+github.setup        marshall ${_name} 0.6.0 v
+
+name                py-${_name}
+categories-append   devel
+license             Apache-2
+maintainers         {gmail.com:yan12125 @yan12125} openmaintainer
+description         A colorful logcat
+long_description    \
+    A colorful and highly configurable alternative to the standard "adb logcat" \
+    command from the Android SDK
+platforms           darwin
+
+python.versions     27
```

Comment:
Note that Python27 isn't going to be supported relatively soon. If you like this port enough, I'd spend the several minutes porting it to Python 3 and send the patches upstream.

Reply:
I submitted one in 2016, but seems upstream is not interested in it :/
https://github.com/marshall/logcat-color/pull/24

Reply:
Well, when the time comes you'll have that available at least.